tests/run/libcpp_all.pyx

```diff
@@ -4,6 +4,7 @@ import cython
 
 cimport libcpp
 
+# cimport libcpp.atomic
```

Comment:
Is there a reason these are added and commented out? If it's a question of requiring a new(er) std, just don't include them here.